src/components/seekbar/seekbar.js

```diff
@@ -554,7 +547,7 @@ class SeekBar extends Component {
    */
   render(props: any, state: Object): React$Element<any> {
     const virtualProgressWidth = `${(props.virtualTime / props.duration) * 100}%`;
-    const progressWidth = `${props.forceFullProgress ? 100 : (props.currentTime / props.duration) * 100}%`;
+    const progressWidth = `${(props.currentTime / props.duration) * 100}%`;
```

Comment:
Consider leaving forceFullProgress as it might fix a bad experience when DVR is very small

Reply:
done